### ISSUE TITLE
Grey 10, disabled toggle needs grey bkgd

### ISSUE THREAD

**Issue by heatherco** [carbon-design-system/carbon#2795]
Grey 10/ big/ item/ disabled on needs a grey 10 background. Currently white background on white tile.

Describe in detail the issue you're having. Please include whether or not this is part of Sketch Libraries or the larger Sketch Kit file.

> What version of the Carbon Design System Kit are you using?
10

> Is this issue related to a specific component? If so please give the page, artboard, and layer/symbol name.
<img width="802" alt="image" src="https://user-images.githubusercontent.com/12818623/57897104-d7b92600-7821-11e9-8a37-7cf0d731cb2f.png">

> What did you expect to happen? What happened instead? What would you like to see changed?
toggle should have a grey 10 background. The disabled-**off** is appearing as it should. It is the disable-**on** that needs to have grey background

**Comment by laurenmrice** [comment]
@heatherco Good catch! This will could potentially be resolved in the future by applying inline themeing, which our devs are in the process of developing.
